_pdf.yml

```diff
@@ -52,7 +52,7 @@ pdf:
     author: 'National Institute of Standards and Technology'
     subject: "This document establishes a standard for a Personal Identity Verification (PIV) system that meets the control and security objectives of Homeland Security Presidential Directive-12. It is based on secure and reliable forms of identity credentials issued by the Federal Government to its employees and contractors. These credentials are used by mechanisms that authenticate individuals who require access to federally controlled facilities, information systems, and applications. This Standard addresses requirements for initial identity proofing, infrastructure to support interoperability of identity credentials, and accreditation of organizations and processes issuing PIV credentials."
     language: 'en-US'
-    lineno: true
+    lineno: false
```

Comment:
@regenscheid I noticed the final FIPS documents all had their line numbers turned off, we'll want to do the same for the non-draft version, right?